src/main/kotlin/no/nb/mlt/wls/application/hostapi/item/ApiItemPayload.kt

```diff
@@ -73,15 +72,13 @@ data class ApiItemPayload(
     @Schema(
         description = """Where the item is located, can be used for tracking item movement through storage systems.""",
         examples = ["SYNQ_WAREHOUSE", "AUTOSTORE", "KARDEX"],
-        accessMode = READ_ONLY,
         required = false
     )
     val location: String?,
     @Schema(
         description = """Quantity on hand of the item, this easily denotes if the item is in the storage or not.
                 If the item is in storage then quantity is 1, if it's not in storage then quantity is 0.""",
         examples = [ "0", "1"],
-        accessMode = READ_ONLY,
         required = false
     )
     val quantity: Int?
```

Comment:
Can probably mark these as non-nullable
```suggestion
    val quantity: Int
```

Reply:
Should also do the same in the notification payload